src/generate-elements.ts

```diff
@@ -85,7 +86,7 @@ function serializeElement(
   const events = resolvedElement.events.map(
       e => ({
         name: e.name,
-        description: e.description,
+        description: e.description || '',
```

Comment:
Client code for elements.json is potentially really diverse, it's not just nodejs and not just code that we write, so I wanted to make it as easy to consume as possible. IMO defaulting to a description of empty string is the easiest thing for a user to consume.

Agreed on generally preferring nulls to undefined to represent a 'not present' value on a field.